_maps/configs/SYN-C_courage-VOID.json

```diff
@@ -0,0 +1,19 @@
+{
+	"map_name": "Courage-class Strike Dropship",
+	"map_short_name": "Courage-class",
+	"prefix": "SYN-C",
+	"map_path": "_maps/shuttles/shiptest/courage.dmm",
+	"map_id": "courage_class",
+	"job_slots": {
```

Comment:
“limit”: 1,

If you’re going to attempt to get a ship this strong merged at least put a cap on how many can spawn.

Reply:
Good to know, sorry for not including that.